cider/util/sqldefs.h

```diff
@@ -321,7 +321,7 @@ inline SqlStringOpKind name_to_string_op_kind(const std::string& func_name) {
   if (func_name == "REPEAT") {
     return SqlStringOpKind::REPEAT;
   }
-  if (func_name == "||") {
+  if (func_name == "||" || func_name == "CONCAT") {
```

Comment:
please refactor this function to a hashmap: std::string -> SqlStringOpKind

Reply:
updated